spec.html

```diff
@@ -2184,42 +2184,55 @@ <h1>
           <h1>
             Number::toString (
               _x_: a Number,
+              _radix_: an integer in the inclusive interval from 2 to 36,
             ): a String
           </h1>
           <dl class="header">
             <dt>description</dt>
-            <dd>It converts _x_ to String format.</dd>
+            <dd>It represents _x_ as a String using a positional numeral system with radix _radix_. The digits used in the representation of a number using radix _r_ are taken from the first _r_ code units of *"0123456789abcdefghijklmnopqrstuvwxyz"* in order. The representation of numbers with magnitude greater than or equal to *1*<sub>𝔽</sub> never includes leading zeros.</dd>
```

Comment:
FYI this spelling of "zeros" is different from what we had before but I'm unable to determine what the OED says, if anything, about the preferred spelling. I will change it back to "zeroes" for now.